local/bin/py/build/actions/workflows.py

```diff
@@ -37,7 +37,7 @@ def workflows(content, content_dir):
                         data = json.loads(f.read())
                     except:
                         logger.warn(f"Error parsing {file_name}")
-            if data and data.get('stability', '') == 'stable':
+            if data and data.get('stability', '') == 'stable' and not data.get('internal') and not data.get('hidden') and not data.get('deprecated'):
```

Comment:
Is this needed? Currently it's set up to only publish if it finds a `stability` of `stable`. For child actions (line 44), it's set up to only publish child actions that do not list a `stability` or are set to `stable`. My understanding is that any `stability` outside of `stable`/null is covered by those rules.

Reply:
@natashadada explained this one to me. Thanks Natasha!